tests/handlers/test_agent.py

```diff
@@ -30,3 +34,28 @@ def test_post_agent_unauthenticated(test_client, db_session):
 
     assert response.status_code == 403
     assert response.json()['detail'] == "Permission denied."
+
+
+def test_list_agent_success(db_session, test_client, session_cookie):
+    response = test_client.get('/agent', cookies=session_cookie)
+
+    assert response.json() == []
+
+    agent = Agent(name='test', oauth_client=OAuthClient(type='confidential'))
+    db_session.add(agent)
+    db_session.commit()
+
+    response = test_client.get('/agent', cookies=session_cookie)
+
+    assert response.status_code == 200
+    response_body = response.json()
+
+    assert len(response_body) == 1
+    assert response_body[0]['name'] == 'test'
```

Comment:
I'd assert more concretely. Most of the response is now unasserted.

I'd also test a list call with more than 1 item I think just for sanity.